recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/start_doma.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/sh
+
+#echo "Waiting for backends"
```

Comment:
Either remove all commented-out `echo` or add function `debug()` and verbose flag.